subsys/bluetooth/common/Kconfig

```diff
@@ -172,6 +172,7 @@ config BT_BUF_CMD_TX_SIZE
 
 config BT_BUF_CMD_TX_COUNT
 	int "Number of HCI command buffers"
+	default 3 if BT_EXT_ADV && BT_CENTRAL
```

Comment:
Just leaving a comment here that `le_ext_adv_param_set` uses 2 HCI Command Buffers in a nested format, starving the connection create cancel when invoked from system work queue to fail silently (This can happen if a pre-emptive thread is starting advertising).

The nested use of HCI Command Buffers will be fixed in https://github.com/zephyrproject-rtos/zephyr/pull/83774/commits/b21c3867a8f5f884c3e5f764badc072b05da8111